lib/app_web/templates/layout/root.html.heex

```diff
@@ -2,6 +2,15 @@
 <html lang="en">
   <head>
     <meta name="csrf-token" content={csrf_token_value()} />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' http://cdnjs.cloudflare.com; script-src-elem 'self' 'unsafe-inline' 'unsafe-eval' http://cdnjs.cloudflare.com "
+    />
```

Comment:
Do we need to add a content security policy for cloudflare?

Reply:
Yep. that's the error we are seeing in: https://github.com/dwyl/mvp/issues/287#issuecomment-1418922531
Adding both as `cloudflare` is the one blocking this ... 🙄

Reply:
Very happy to remove it if you think this will work without allowing `cloudflare` access ... 🤷‍♂️ 
But ultimately they are the gatekeepers. 🙃

Reply:
Didn't see the error was referencing cloudflare, we can try with it, not sure if it's required or if it can work by adding only the plausible one